static/elements/chromedash-myfeatures-page.js

```diff
@@ -71,14 +71,15 @@ export class ChromedashMyFeaturesPage extends LitElement {
     openApprovalsDialog(this.user.email, e.detail.feature);
   }
 
-  renderBox(title, query, columns, opened=true) {
+  renderBox(title, query, columns, sortSpec='', opened=true) {
```

Comment:
Just to confirm, this functionality only works for the approval/review tables? Is there any reason why this shouldn't also work with the "Features I can edit" or "Features I starred" tables?

Reply:
We could specify a sorting order for any of the feature tables.   The default is to sort by `-created.when`, so newly created feature entries are on top and things never change order.  That seems like a good default for the other boxes.